### ISSUE TITLE
Suivi inscriptions (PipeDream)

### ISSUE THREAD

**Issue by Mathieu-CdlT** [chemins-de-la-transition/semapps#529]
Hello, 

Sur les inscriptions, on a sur la plateforme la possibilité de renvoyer des utilisateurs vers des liens d'inscriptions externes (formulaires d'inscription de nos partenaires). Or qd on clique pour s'inscrire et qu'on est renvoyé vers un lien externe, on n'a pas nous de traçabilité de qui contacte qui (pas d'incrémentation du fichier Pipedream). Pourrait-on mettre en place dans ce cas aussi ? On a pas mal de liens externes sur les événements qu'on référence et besoin de pouvoir savoir l'impact pour nos partenaires. 

Merci merci :)! 

Exemple :  https://lescheminsdelatransition.org/Event/https%3A%2F%2Fdata.lescheminsdelatransition.org%2Fevents%2Fjournees-d-initiation-a-la-culture-du-houblon-en-agriculture-biologique/show

**Comment by Mathieu-CdlT** [comment]
Par ailleurs, quelle possibilité qu'on mail soit envoyé au partenaire en lui disant "un.e tel vient a essayé de s'inscrire sur votre formulaire d'inscription via les Chemins de la Transition"? @srosset81 @GuillaumeAV

**Comment by srosset81** [comment]
Si c'est un lien externe inconnu, tout ce qu'on pourrait "tracker", c'est le clic, sans aucune assurance que l'utilisateur se soit vraiment inscrit. Et comme j'ai dit ailleurs, je pense que ce serait mieux de suivre ce genre de données via Matomo directement.

**Comment by srosset81** [comment]
Voir https://fr.matomo.org/faq/reports/implement-event-tracking-with-matomo/

**Comment by Mathieu-CdlT** [comment]
Oui, sauf que sur Matomo si j'ai bien compris (et ok c'est pas mon rayon), on ne sait pas les noms / prénoms des gens qui contactent. Du coup, ca rend impossible la réalisation de points avec nos partenaires pour savoir si ils ont eu des inscrits de la part des CdlT. Et donc sur notre modèle éco de participation libre des partenaires. C'est en particulier pour ces raisons qu'on avait décidé de rendre la création d'un profil utilisateur obligatoire pour pouvoir contacter une initiative / s'inscrire sur un événement.

**Comment by srosset81** [comment]
Non c'est possible de connaître l'ID de l'utilisateur qui génère l'événement (clic):
https://fr.matomo.org/faq/reports/set-up-user-id-tracking-in-matomo/

**Comment by srosset81** [comment]
Fait
Visible [ici](https://docs.google.com/spreadsheets/d/1MddTSnKZUIzZo4opaDUF_TKB40IMuJSPSN7wQ_ARCHM/edit#gid=426417449) (onglet "Clics")
